pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -293,7 +293,7 @@ func init() {
 		}
 		cloud, err := NewOpenStack(cfg)
 		if err != nil {
-			klog.V(1).Infof("New openstack client created failed with config")
+			klog.Warningf("New openstack client created failed with config: %s", err)
```

Comment:
s/ %s / %v

Reply:
`			klog.Warningf("New openstack client created failed with config: %v", err)`

Reply:
Out of curiosity, why `%v`? Is it in case of  `err` being of a type other than a `string`?

Reply:
Thanks, done.